### PR TITLE
Integrating new version of react-discovery-ui to fix issue with visualization page

### PR DESCRIPTION
**Pull request by LtChae**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7886,7 +7886,7 @@
     },
     "external-editor": {
       "version": "2.2.0",
-      "resolved": "https://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/external-editor/-/external-editor-2.2.0.tgz",
```
Comment (jmorris-pillar) [inline package-lock.json]: this has been reverted to http, can we not make this change

File: package-lock.json [hunk 2]
```diff
@@ -10832,7 +10813,7 @@
     },
     "inquirer": {
       "version": "5.2.0",
-      "resolved": "https://registry.npmjs.org/inquirer/-/inquirer-5.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/inquirer/-/inquirer-5.2.0.tgz",
```
Comment (jmorris-pillar) [inline package-lock.json]: see above

File: package-lock.json [hunk 3]
```diff
@@ -19561,7 +19542,7 @@
     },
     "simple-html-tokenizer": {
       "version": "0.1.1",
-      "resolved": "https://registry.npmjs.org/simple-html-tokenizer/-/simple-html-tokenizer-0.1.1.tgz",
+      "resolved": "http://registry.npmjs.org/simple-html-tokenizer/-/simple-html-tokenizer-0.1.1.tgz",
```
Comment (jmorris-pillar) [inline package-lock.json]: see above

File: package-lock.json [hunk 4]
```diff
@@ -10207,7 +10188,7 @@
     },
     "html-webpack-plugin": {
       "version": "3.2.0",
-      "resolved": "https://registry.npmjs.org/html-webpack-plugin/-/html-webpack-plugin-3.2.0.tgz",
+      "resolved": "http://registry.npmjs.org/html-webpack-plugin/-/html-webpack-plugin-3.2.0.tgz",
```
Comment (ManApart) [inline package-lock.json]: Same for these next 3

Comment (LtChae) [reply package-lock.json]: Fixed. @jmorris-pillar verify

**Pull request merged**